aws_lambda_builders/workflows/custom_make/make.py

```diff
@@ -92,9 +93,11 @@ def run(self, args, env=None, cwd=None):
 
         # Log every stdout line by iterating
         for line in p.stdout:
-            decoded_line = line.decode("utf-8").strip()
-            LOG.info(decoded_line)
+            sys.stderr.buffer.write(line)
```

Comment:
AFAIK, `sys.stderr.buffer.write` won't include a newline at the end of the string which might make the next log statement messy. Should we add a newline after?

Reply:
ref: https://docs.python.org/3/library/io.html#io.IOBase.readline

For the line that gets returned by the iterator (`for line in stdout`), it calls `readline()` under the hood and there should always be a new line character at the end of each line. So here, we should be fine to not include a new line of our own.

Reply:
Awesome, thanks for digging in!

Reply:
just can you add a comment why we used sys.stderr to log the terraform output instead of log.info.